client/js/components/GameView/Card.vue

```diff
@@ -21,10 +21,10 @@
       class="valid-move target-overlay"
       opacity=".8"
     />
-    <transition name="slide-above">
+    <transition :name="scuttledByTransition">
       <template v-if="scuttledBy">
         <img
-          class="scuttled-by-card"
+          :class="scuttledByClass"
```

Comment:
I don't have a much better way to suggest doing this, but I don't love moving these to computed props as it makes it harder to reference. But considering it's a 3 way selection, I can admit that we need some sort of structure.

Reply:
Harder to reference than in-lining the class? Like 
```suggestion
          :class="{"scuttled-by-card scuttled-by-opponent": this.controlledBy === 'player', "scuttled-by-card scuttled-by-opponent": this.controlledBy === 'opponent'}
```